adserver/models.py

```diff
@@ -1222,6 +1233,18 @@ def show_on_domain(self, url):
 
         return True
 
+    def show_to_niche_targeting(self, weights):
+        """Filter flights based on distance of niche targeting"""
+
+        if self.niche_targeting:
+            if self.campaign.advertiser in weights:
+                goal_weight = self.niche_targeting
+                return weights[self.campaign.advertiser] < goal_weight
+            else:
+                return False
```

Comment:
```suggestion
            return False
```

Reply:
I really don't love this linting rule -- I find the explicit else much more readable, but alas.